environment_cuda.yml

```diff
@@ -0,0 +1,15 @@
+name: mtenn
```

Comment:
does it make sense to give this a different name? not sure if there's any value to having both environments set up